packages/edit-post/src/style.scss

```diff
@@ -93,15 +93,6 @@ body.js.block-editor-page {
 
 @include wordpress-admin-schemes();
 
-.interface-interface-skeleton__header .edit-post-header {
-	@supports (scrollbar-gutter: stable) {
-		// The scrollbar-gutter property ensures space is reserved for the scrollbar to appear,
-		// when scrollbars are set to be always visible. This ensure icons stay visually aligned.
-		scrollbar-gutter: stable;
-		overflow: hidden;
-	}
-}
-
```

Comment:
I don't understand the connection between the two changes. If the 10 px extra width solves the focus ring issue why do we need to remove this tweak? As far as I know this was added to prevent thr ugly jumping around when scrollbars appear.

Reply:
The scrollbars in the sidebar are defined [elsewhere](https://github.com/WordPress/gutenberg/blob/e2237b8ffd98c3211af6bbff3b642fb9682e29c8/packages/edit-post/src/style.scss#L115) and are not affected by removing this CSS. This CSS affects the header. The following screencast might help you understand why this style must be removed.

We could take this scrollbar width into account when calculating the width of the fixed toolbar, but this is a bit tricky. See ChatGPT's answer for more details 😅: https://chat.openai.com/share/08933117-0064-4de0-ae71-747550c00c55

https://github.com/WordPress/gutenberg/assets/54422211/192ec2a5-08ae-4e66-97e4-6aa7b6a02bb6